js/client-teambuilder.js

```diff
@@ -200,11 +235,33 @@
 			Storage.nwLoadTeams();
 		},
 		edit: function(i) {
+			if (this.curmoving !== false) return;
 			var i = +i;
 			this.curTeam = teams[i];
 			this.curTeamIndex = i;
 			this.update();
 		},
+		"move": function(i) {
+			if (this.curmoving !== false) return;
```

Comment:
Any particular reason why these aren't written `if (this.curmoving === true) return;` ?